plugins/env/app/views/samson_env/_project_form.html.erb

```diff
@@ -15,11 +15,11 @@
     %>
   <% end %>
 
-  <% if defined?(EnvironmentVariable) && folder = EnvironmentVariable.config_service_folder(@project, required: false) %>
+  <% if defined?(EnvironmentVariable) && folder = EnvironmentVariable.config_service_folder(@project) %>
     <%= form.input(
         :config_service,
         label: "Use Config service",
-        help: "Use config service to manage deployment environment variables, base url is #{folder}"
+        help: "Use config service to manage deployment environment variables, S3 path is #{folder}"
```

Comment:
should show the bucket and the folder to be obvious ?
something like `#{bucket}:#{folder}`

Reply:
I can use the whole S3 path, like `s3://bucket-name/folder/path`